src/clj/rems/application/commands.clj

```diff
@@ -413,6 +421,33 @@
                                           (when-let [end (:entitlement-end cmd)]
                                             {:entitlement/end end})))))
 
+(defn- empty-redacted-attachments-error [cmd]
+  (when (empty? (:redacted-attachments cmd))
+    {:errors [{:type :empty-redacted-attachments}]}))
+
+(defn- find-original-attachment-event-ids
```

Comment:
Should we use this information to allow the decider, reviewer to only redact attachments they added, but allow handler to redact any attachment?